src/parser/bison_parser.y

```diff
@@ -215,6 +218,7 @@ int yyerror(YYLTYPE* llocp, SQLParserResult* result, yyscan_t scanner, const cha
 %type <group_t>		    opt_group
 %type <alias_t>		    opt_table_alias table_alias opt_alias alias
 %type <with_description_t>  with_description
+%type <import_type_t>	opt_file_type import_file_type
```

Comment:
Could you harmonize the mix of `file_type`, `import_file_type`, and `import_type`?

Reply:
I myself would prefer using FileType instead of ImportType in general for both Import and Export statement (so it would sense for both). I just thought renaming it could be a thing regarding downwards compatibility.

Reply:
Ok, in that case, please add a comment.

Reply:
Do I get it right and renaming `ImportType` to `FileType` and altering its values to `kFileCSV`, ... with just adding a comment there that this has been done is okay?

Reply:
My thought was to leave it as it, but add comments where appropriate that this is done for compat reasons

Reply:
Alright, are those two comments enough?